homework/Contributor.js

```diff
@@ -13,7 +13,37 @@ class Contributor {
    * @param {HTMLElement} container The container element in which to render the contributor.
    */
   render(container) {
-    // TODO: replace the next line with your code.
-    Util.createAndAppend('pre', container, JSON.stringify(this.contributor, null, 2));
+    // Util.createAndAppend('pre', container, JSON.stringify(this.contributor, null, 2));
```

Comment:
I saw many commented code in your files, and it's fine for now, but in the near future you will see thousends of lines, and you will probably think about leaving it out.